keys.txt

```diff
@@ -648,8 +651,8 @@ district    // used in Liberia with healthcare, address-based
 sub_district    // used in Liberia with healthcare, address-based
 
 us.fo:Adressutal    // unspecified integer, see https://wiki.openstreetmap.org/wiki/Import/Catalogue/usfo , automatic translation from Danish says "Address number"; used for import syncing - see https://www.openstreetmap.org/changeset/1888617
-us.fo:Galdandi_frá  // discardable, see https://wiki.openstreetmap.org/wiki/Import/Catalogue/usfo , asked on https://www.openstreetmap.org/changeset/1888617 (means "valid from" date)
```

Comment:
weird that also that line got changed

Reply:
Arg that's an encoding issue. I edited via GitHub, which converted to UTF-8 I think